Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -47,9 +47,10 @@ export function processSonarQubeIntegration(sqBuildFolder:string):Q.Promise<void
     // Wait for all promises to complete before proceeding (even if one or more promises reject).
     return Q.allSettled([
```

Comment:
Could you call the processSQBuildSummary directly ? Q.allSettled does not seem required